packages/perseus/src/widgets/expression/__docs__/expression.stories.tsx

```diff
@@ -71,6 +75,12 @@ export const ExpressionItem3: Story = {
     },
 };
 
+export const ExpressionItem4: Story = {
+    args: {
+        item: expressionItem4,
+    },
+};
+
```

Comment:
Can you add a doc comment above this story explaining how it can be used to test division? The comment should show up above the story in Storybook as a description.